configs/defs/fallback.textproto

```diff
@@ -6477,6 +8485,55 @@ benchmark_configs: {
       "}}"
   }
 }
+benchmark_configs: {
+  id: "ssh-x11-forwarding-disabled-fallback"
+  compliance_note: {
+    version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    title: "Ensure SSH X11 forwarding is disabled"
+    description:
+      "The X11Forwarding parameter provides the ability to tunnel X11 traffic "
+      "through the connection to enable remote graphic connections."
+    rationale:
+      "Disable X11 forwarding unless there is an operational "
+      "requirement to use X11 applications directly. "
+      "There is a small risk that the remote X11 servers of users who are logged in via "
+      "SSH with X11 forwarding could be compromised by other users on the X11 server. "
+      "Note that even if X11 forwarding is disabled, users can always install their own forwarders."
+    remediation:
+      "Edit the /etc/ssh/sshd_config file to set the parameter as follows: \n"
+      "```X11Forwarding no```"
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/etc/ssh/sshd_config\"}}"
```

Comment:
Add a check alternative for /etc/ssh/sshd_config not existing - also for the checks below